.bumpversion.cfg

```diff
@@ -0,0 +1,8 @@
+[bumpversion]
```

Comment:
Why a cfg file in a node project? There is already the version in the package.json file. This appears to be a python specific solution.

Reply:
If you want a tool to bump the project's sem ver, please use a node solution like `semver` https://www.npmjs.com/package/semver

Reply:
Nope. We use `bumpversion` as our release tool from DevOps everywhere. Having a single interface across all projects is valuable.

Reply:
as an open-source I would go with both :)

Reply:
I gave this a whirl... `semver` doesn't bump files or understand `package.json`; it only understands versions, so not useful enough for these purposes.